ppr-ui/src/store/state/state-model.ts

```diff
@@ -380,6 +380,14 @@ export const stateModel: StateModelIF = {
       phoneNumber: '',
       phoneExtension: ''
     },
+    qsReviewConfirm: {
+      isRequirementsConfirmed: false,
+      authorization: {
+        isAuthorizationConfirmed: false,
+        legalName: '',
+        date: ''
+      }
+    },
```

Comment:
I'm not sure we need to wrap these component properties in `qsReviewConfirm`, unless there is some good value to do so?

Reply:
I set localState, and use setters to deal with modifying computed primitive value. 
As we discussed in the future if we take the time to learn and use `state` instead of `getters` in store we can probably remove a lot of the extra code.

If you can please just take a quick look and let me know, if you see any errors or further possible clean up.

Reply:
You bet, looks fine to me 👍